main/windows/frames/viewInstances.ts

```diff
@@ -26,38 +26,123 @@ export default {
     const viewInstance = createViewInstance(view.ens)
     const { session } = extract(view.url)
 
-    viewInstance.webContents.session.webRequest.onBeforeSendHeaders((details, cb) => {
-      if (!details || !details.frame) return cb({ cancel: true }) // Reject the request\
-
-      const appUrl = details.frame.url
-
-      if (
-        // Initial request for app
-        details.resourceType === 'mainFrame' &&
-        details.url === view.url &&
-        !appUrl
-      ) {
-        return cb({ requestHeaders: details.requestHeaders }) // Leave untouched
-      } else if (
-        // devtools:// request
-        details.url.startsWith('devtools://')
-      ) {
-        return cb({ requestHeaders: details.requestHeaders }) // Leave untouched
-      } else if (
-        // Reqest from app
-        appUrl === view.url
-      ) {
-        const { ens, session } = extract(appUrl)
-        if (ens !== view.ens || !server.sessions.verify(ens, session)) {
-          return cb({ cancel: true })
-        } else {
-          details.requestHeaders['Origin'] = view.ens
-          return cb({ requestHeaders: details.requestHeaders })
-        }
-      } else {
-        return cb({ cancel: true }) // Reject the request
-      }
-    })
+    if (!viewInstance) return
+
+    // const animationSpeed = 8 // Pixels per frame
```

Comment:
can we remove some of these comments or is it likely we'll need them in the near future?

Reply:
yeah all the comments i left in need to be uncommented